README.md.template

```diff
@@ -1,10 +1,24 @@
-# Bazelrio
-Bazelrio is a collection of [bazel](http://bazel.build) utilities for FRC.
+<h1 align="center">
+    <img src="https://avatars.githubusercontent.com/u/84363228?s=200&v=4" width="256">
+    <br />
+    BazelRIO
+    <br />
+    <img src="https://img.shields.io/badge/semver-0.1.0-blue">
```

Comment:
It would be sick if this could be replaced with the release tag

Reply:
if we do this let's also use release tags for the download url (ie https://github.com/bazelRio/bazelRio/archive/refs/tags/v0.1.0.zip). makes things a little more readable for the user.